### PR TITLE
Avoid listing views in memory connector listTables

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

**Comment by findepi** [comment]
Did we settle on the decision already?

**Comment by raunaqmorarka** [comment]
> Did we settle on the decision already?

It appeared to me that we were leaning this way, if not, we can certainly continue to discuss it further.

**Comment by findepi** [comment]
cc @trinodb/maintainers

File: core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java [hunk 1]
```diff
@@ -199,7 +199,7 @@ default ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connec
     }
 
     /**
-     * List table and view names, possibly filtered by schema. An empty list is returned if none match.
+     * List table names, possibly filtered by schema. An empty list is returned if none match.
```
Comment (sopel39) [inline core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java:202]: Hive connector also returns views as part of `listTables` call.
If we decide we go this way, we should add `listRelations` method too

**Pull request closed without merging**